[2022-03-15 19:33:54,892] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: trigger_rule.task_2 manual__2022-03-15T19:33:51.377489+00:00 [queued]>
[2022-03-15 19:33:54,994] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: trigger_rule.task_2 manual__2022-03-15T19:33:51.377489+00:00 [queued]>
[2022-03-15 19:33:54,996] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 19:33:54,996] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-03-15 19:33:54,996] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 19:33:55,123] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): task_2> on 2022-03-15 19:33:51.377489+00:00
[2022-03-15 19:33:55,125] {standard_task_runner.py:52} INFO - Started process 12802 to run task
[2022-03-15 19:33:55,139] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'trigger_rule', 'task_2', 'manual__2022-03-15T19:33:51.377489+00:00', '--job-id', '159', '--raw', '--subdir', 'DAGS_FOLDER/trigger_rule.py', '--cfg-path', '/tmp/tmp5p4_sjog', '--error-file', '/tmp/tmp4gy2gys9']
[2022-03-15 19:33:55,140] {standard_task_runner.py:77} INFO - Job 159: Subtask task_2
[2022-03-15 19:33:55,547] {logging_mixin.py:109} INFO - Running <TaskInstance: trigger_rule.task_2 manual__2022-03-15T19:33:51.377489+00:00 [running]> on host airflowvm
[2022-03-15 19:33:56,383] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=trigger_rule
AIRFLOW_CTX_TASK_ID=task_2
AIRFLOW_CTX_EXECUTION_DATE=2022-03-15T19:33:51.377489+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-03-15T19:33:51.377489+00:00
[2022-03-15 19:33:56,387] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-03-15 19:33:56,396] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 30']
[2022-03-15 19:33:56,734] {subprocess.py:85} INFO - Output:
[2022-03-15 19:34:26,745] {subprocess.py:93} INFO - Command exited with return code 0
[2022-03-15 19:34:26,794] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=trigger_rule, task_id=task_2, execution_date=20220315T193351, start_date=20220315T193354, end_date=20220315T193426
[2022-03-15 19:34:32,774] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-15 19:34:32,941] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
